api/Attr.json

```diff
@@ -141,7 +141,8 @@
             "experimental": false,
             "standard_track": true,
             "deprecated": false
-          }
+          },
+          "spec_url": "https://dom.spec.whatwg.org/#dom-attr-name"
```

Comment:
I'm not in a position (yet) to review the individual entries here, but I do want to highlight a structural problem: `spec_url` ought to be a sibling of `support` and `status` (usually coming before both, right after the `__compat` line).

Reply:
It is a sibling of support and status? (It's at the end because, uh, it got added programmatically and as is often the case with JS dictionaries insertion order being used. I guess I could change the script to add it at the beginning.)

Reply:
Sorry, I misread the nesting. 🤦 You're right, it's at the right place, but it would be nice to see it at the top (I realize that the actual representation is unordered, but it's nice to have a specific order in the source).